serialize_corpora.py

```diff
@@ -2,7 +2,7 @@
 
 from transformers import HfArgumentParser
 
-from src.processing import (
+from src.processor import (
```

Comment:
What is the reason for changing the name of module and the variable in script? to: @bzantium

Reply:
This is because to be consistent with collator. Moreover, transformers has the same naming style. The plural form might be better though.

Reply:
I got it.